Examples/Algorithms/Fatras/src/FatrasSimulation.cpp

```diff
@@ -237,16 +239,36 @@ ActsExamples::ProcessCode ActsExamples::FatrasSimulation::execute(
   ACTS_DEBUG(simHitsUnordered.size() << " simulated hits");
 
   // order output containers
+#if BOOST_VERSION >= 107800
```

Comment:
Just for reference: I think in Core we shouldn't this but in the examples it's absolutely fine!